insights/config/specs.py

```diff
@@ -410,6 +411,7 @@
     "iface"                     : "/sbin/ip -o link | awk -F ': ' '/.*link\\/ether/ {print $2}'",
     "md5chk_files"              : "/bin/ls -H /usr/lib*/{libfreeblpriv3.so,libsoftokn3.so} /etc/pki/product*/69.pem /etc/fonts/fonts.conf /dev/null 2>/dev/null",
     "module"                    : "/bin/ls /sys/module",
+    "java_command_package"      : "for jp in `/bin/ps auxwww | grep java | grep -v grep| awk '{print $11}' | sort -u`; do echo $jp `readlink -e $jp | xargs rpm -qf`; done",
```

Comment:
Since there are two parts for the `echo` in the pre_command, better add quotes to surround the two parts.

``echo "$jp `readlink -e $jp | xargs rpm -qf`"``

Reply:
As we discussed, will not change it.